lib/systems/inspect.nix

```diff
@@ -8,6 +8,10 @@ let abis = lib.mapAttrs (_: abi: builtins.removeAttrs abi [ "assertions" ]) abis
 
 rec {
   patterns = rec {
+    # NOTE: normal stuff should use i686, this type is purely for
+    # "i386-pc-elf" target (e.g. coreboot and its payloads)
+    isi386         = { cpu = cpuTypes.i386; };
+
     isi686         = { cpu = cpuTypes.i686; };
```

Comment:
So I think we should have `isx86_32` and `isx86_64`, with `isi686` as an alias for `isx86_32`, perhaps to be deprecated. This matches what we do for arm. It also avoids a situation where we falsely say `i386` isn't `i686`, which ignores subtyping / compatibility.

Reply:
That sounds good! If anything truly works on just i686 then just ask users to do ```stdenv.hostPlatform.parsed.cpu.name == "i686"```.

Reply:
LLVM does http://llvm.org/doxygen/Triple_8cpp_source.html has `i386` through `i986`, fwiw.

Reply:
I think we should limit it to i386, i486, i586, and i686. The newer ones never really caught on.